client/src/common/components/Members/components/MemberDetails.jsx

```diff
@@ -358,6 +383,59 @@ class MemberDetails extends PureComponent {
     );
   };
 
+  renderLeaveOption = () => {
+    const {
+      _id: userId,
+      currentUser: { id: currentUserId },
+      onClose,
+      route
+    } = this.props;
+    const { isLeaveConfirmationVisible, pendingForLeaving } = this.state;
+
+    if (userId === currentUserId) {
```

Comment:
make an early return instead of putting all this code in if block.
```
if (userId !== currentUserId) {
    return null;
}

return (
   <div...
```